src/lib/worldGenerations/grass_field.js

```diff
@@ -2,13 +2,15 @@ const Vec3 = require('vec3').Vec3
 
 function generation ({ version }) {
   const Chunk = require('prismarine-chunk')(version)
+  const mcData = require('minecraft-data')(version)
 
   function generateSimpleChunk () {
     const chunk = new Chunk()
 
     for (let x = 0; x < 16; x++) {
       for (let z = 0; z < 16; z++) {
-        chunk.setBlockType(new Vec3(x, 50, z), 2)
+        chunk.setBlockType(new Vec3(x, 50, z), mcData.blocksByName.grass_block.id)
```

Comment:
Careful, in old version grass_block is grass

Reply:
Do you know which version ?

Reply:
Bellow 1.13

Reply:
Do you recommend to use serv.supportFeature() or a normal condition ?

Reply:
serv.supportFeature would be good, I believe we already have something similar in the code